tools/kvstore-test/tests/light-client.rs

```diff
@@ -74,23 +74,18 @@ fn make_supervisor() -> Supervisor {
     let node_address: rpc::Url = "http://127.0.0.1:26657".parse().unwrap();
 
     let options = light_client::Options {
-        trust_threshold: TrustThreshold {
-            numerator: 1,
-            denominator: 3,
-        },
+        trust_threshold: TrustThreshold::new(1, 3).unwrap(),
```

Comment:
Just a minor suggesiton - if you like we could use `Default::default()` here. Also, the `impl default for TrustThresholdFraction` could reuse the `ONE_THIRD` const.

Reply:
Done